radio/src/CMakeLists.txt

```diff
@@ -99,6 +99,12 @@ else()
   message(FATAL_ERROR "Unknown PCB '${PCB}'")
 endif()
 
+# Add RADIO_xxx definition
+string(TOUPPER ${FLAVOUR} RADIO_NAME)
+string(REPLACE "+" "P" RADIO_NAME ${RADIO_NAME})
+string(CONCAT RADIO_NAME "RADIO_" ${RADIO_NAME})
+add_definitions(-D${RADIO_NAME})
```

Comment:
It would be nice to define `${RADIO_NAME}` as a CMake variable to be able to do things like:
```cmake
if(RADIO_XYZ)
  # do something special
endif()
```

Reply:
Done.